### ISSUE TITLE
Refactor facial expressions to look for FBC animation engine first

### ISSUE THREAD

**Issue by littlesera** [littlesera/LSCG#118]
Intention is to allow FBC engine to take back over once LSCG is finished with an expression.
Also look to clean up over-eager expression sets.

- Define an abstraction for CharacterSetFacialExpression and check if FBC's animation engine is enabled 
  - `window.fbcSettingValue("animationEngine")`
  - use `window.fbcPushEvent` if so

**Comment by littlesera** [comment]
No longer relevant with BC changes and FBC dropping animation system support.
